buildbot_nix/__init__.py

```diff
@@ -239,7 +262,9 @@ def __init__(self, **kwargs: Any) -> None:
     @defer.inlineCallbacks
     def run(self) -> Generator[Any, object, Any]:
         # run `nix build`
-        cmd: remotecommand.RemoteCommand = yield self.makeRemoteShellCommand()
```

Comment:
What was the issue with the previous line?

Reply:
All of your questions, types, typechecking seemed to be broken for me, soemhow through a yield it wouldnt type check, my guess is that a yield can return an error value. I wanted to remove all the type errors so its easier for me to write (and for the future). I can revert these bits no problem if so desired

Reply:
What typechecker are you using?

Reply:
pyright

Reply:
I will try mypy then and clean up these workarounds 👍. Sorry for the mess

Reply:
Alright. You can use `nix fmt` to get a local type check.